driver_cpl/cime_config/config_component.xml

```diff
@@ -644,16 +644,6 @@
     <desc>Operating system - DO NOT EDIT UNLESS for userdefined machine - ignored once Macros has been created.</desc>
   </entry>
 
-  <entry id="COMPILERS">
```

Comment:
My change to allow multiple MPILIBS on a machine was causing an issue in parsing the machines file to write the env_build file - these variables are not used in a case so the best solution was not to write them there.